04_beerhouse/src/MainPage.jsx

```diff
@@ -0,0 +1,208 @@
+import styled from 'styled-components';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import BeerHouse from './BeerHouse';
+
+const getLocation = (errHandler) => {
+  if ('geolocation' in navigator) {
+    return new Promise((resolve) => {
+      navigator.geolocation.getCurrentPosition(
+        (position) => {
+          const {
+            coords: { latitude: y, longitude: x },
+          } = position;
+          resolve({ x, y });
+        },
+        (e) => {
+          alert('HTTPS 연결을 확인해주세요.');
+          errHandler && errHandler();
+        }
+      );
+    });
+  }
+
+  return { x: 126.8350976, y: 37.617664 };
+};
+
+async function 위치가져오기() {
+  const result = await getLocation();
+  const { x, y } = result;
+  return { x, y };
+}
+
+async function 내근처맥주집가져오기() {
+  const currentLocation = await 위치가져오기();
+  const result = await axios.get(
+    'https://dapi.kakao.com/v2/local/search/keyword',
+    {
+      headers: {
+        Authorization: `KakaoAK ${process.env.REACT_APP_KAKAO_AK}`,
+      },
+      params: {
+        x: currentLocation.x,
+        y: currentLocation.y,
+        radius: 1000,
+        query: '맥주',
+      },
+    }
+  );
```

Comment:
```suggestion
  const {x, y}= await 위치가져오기();
  const result = await axios.get(
    'https://dapi.kakao.com/v2/local/search/keyword',
    {
      headers: {
        Authorization: `KakaoAK ${process.env.REACT_APP_KAKAO_AK}`,
      },
      params: {
        x: x,
        y: y,
        radius: 1000,
        query: '맥주',
      },
    }
  );
```
구조분해할당으로 이렇게 해도 좋을 것 같아!